CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## v6.1.1 (04/12/2021)
+
+#### Bug fixes and enhancements:
+
+* [NSTK-2256] Add missing toleration to db pods
```

Comment:
These are internal issue numbers and don't mean anything to customers: we should create issues to represent them or just put NO-ISSUE